docs/CONTRIBUTING.md

```diff
@@ -93,3 +93,12 @@ python setup.py typecheck
 # Code of Conduct
 
 [ONNX Open Source Code of Conduct](http://onnx.ai/codeofconduct.html)
+
+# Instructions for handling Corporate Contributor License Agreements (CCLAs)
+  * Step 1: Download the CCLA from [here]() 
+  * Step 2: Complete the form
+  * Step 3: Open a PR in the https://github.com/onnx/steering-committee/tree/master/CCLAs repo with the completed CCLA attached in the [CCLA directory](https://github.com/onnx/onnx/tree/master/community/CCLAs)
+  * Step 4: The steering committee members will be auto tagged for review including: @jspisak, @prasanthpul, @ciphr, @harryskim and Dilip Sequeira
+  
+  The doc will be reviewed by the, currently forming, legal steering committee made of counsel from each of the steering committer members. 
```

Comment:
Why does legal need to review each one? if it follows the template, then it should be just adding the github IDs to the whitelist.